docs/root/configuration/overview/v2_overview.rst

```diff
@@ -0,0 +1,330 @@
+              port_value: 1234
+
+
+The versioning and type URL scheme that appear above are explained in more
+detail in the `streaming gRPC subscription protocol
+<https://github.com/envoyproxy/data-plane-api/blob/master/XDS_PROTOCOL.md#streaming-grpc-subscriptions>`_
+documentation.
+
+Dynamic
+^^^^^^^
+
+A fully dynamic bootstrap configuration, in which all resources other than
+those belonging to the management server are discovered via xDS is provided
+below:
+
+.. code-block:: yaml
+
+  admin:
+    access_log_path: /tmp/admin_access.log
+    address:
+      socket_address: { address: 127.0.0.1, port_value: 9901 }
+  
+  dynamic_resources:
+    lds_config:
+      api_config_source:
+        api_type: GRPC
+        cluster_name: [xds_cluster]
+    cds_config:
+      api_config_source:
+        api_type: GRPC
+        cluster_name: [xds_cluster]
+
+  static_resources:
+    clusters:
+    - name: xds_cluster
+      connect_timeout: 0.25s
+      type: STATIC
+      lb_policy: ROUND_ROBIN
+      hosts: [{ socket_address: { address: 127.0.0.3, port_value: 5678 }}]
+
+The management server could respond to LDS requests with:
+
+.. code-block:: yaml
+
+  version_info: "0"
+  resources:
+  - "@type": type.googleapis.com/envoy.api.v2.Listener
+    name: listener_0
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          stat_prefix: ingress_http
+          codec_type: AUTO
+          rds:
+            route_config_name: local_route
+            config_source:
+              api_config_source:
+                api_type: GRPC
+                cluster_name: [xds_cluster]
+          http_filters:
+          - name: envoy.router
+
+The management server could respond to RDS requests with:
+
+.. code-block:: yaml
+
+  version_info: "0"
+  resources:
+  - "@type": type.googleapis.com/envoy.api.v2.RouteConfiguration
+    name: local_route
+    virtual_hosts:
+    - name: local_service
+      domains: ["*"]
+      routes:
+      - match: { prefix: "/" }
+        route: { cluster: some_service }
+
+The management server could respond to CDS requests with:
+
+.. code-block:: yaml
+
+  version_info: "0"
+  resources:
+  - "@type": type.googleapis.com/envoy.api.v2.Cluster
+    name: some_service
+    connect_timeout: 0.25s
+    lb_policy: ROUND_ROBIN
+    type: EDS
+    eds_cluster_config:
+      eds_config:
+        api_config_source:
+          api_type: GRPC
+          cluster_name: [xds_cluster]
+
+The management server could respond to EDS requests with:
+
+.. code-block:: yaml
+
+  version_info: "0"
+  resources:
+  - "@type": type.googleapis.com/envoy.api.v2.ClusterLoadAssignment
+    cluster_name: some_service
+    endpoints:
+    - lb_endpoints:
+      - endpoint:
+          address:
+            socket_address:
+              address: 127.0.0.2
+              port_value: 1234
+
+Management server endpoints
```

Comment:
I find this section to be a little confusing. I don't think it's immediately clear to the user what these paths are for or what they mean. Should we somehow map them to whether they are the gRPC streaming endpoint or the REST endpoint and possible link to the protos of where they are defined?

Reply:
Yeah, I've reorged a bit and added proto links. LMK if this improves or not. I also ported some of the ADS stuff across, since it turns out this ended up being a fairly important v2 feature to some (Hello Istio..).